### DISCUSSION TITLE
mail Rejected By Google

### DISCUSSION THREAD

**Discussion by sumoo13** [ltb-project/self-service-password#975] [General]
Hello,

Reset emails sent to gmail addresses are refused because they do not respect RFC 5322
i have a return that says: There are multiple Subject 550-5.7.1 headers
if i delete the value $mail_from_name there is one that is set by default?
any idea what to change? maybe in the php mailer part?

Jean-Philippe

**Comment by coudot** [comment]
You should configure `$mail_smtp_debug` to see how your mail is formatted

// PHPMailer's SMTP debug info level: 0 = off, 1 = commands, 2 = commands and data, 3 = as 2 plus connection status, 4 = low level data output.

https://self-service-password.readthedocs.io/en/stable/config_mail.html#phpmailer

**Reply by sumoo13** [reply]
Hello, 
thank's for you reply, where i cant found the debug, and where can i change the mail format ?
for the moment I modified the sendtokn.php file and I replaced the send_mail function with the php mail function. and I am no longer blocked, it is a problem with phpmailer.
